content/user/advanced-modules/KnowledgeBase.adoc

```diff
@@ -0,0 +1,74 @@
+
+Article text:: Enter a detailed description of the solution.
+If further when entering the topic of a new appeal,
+matches are found with the topic or text of the Knowledge Base article,
+then this article will be automatically offered as a possible solution.
+
+Version of the article:: Specify the version of the article.
+Resolution:: Indicate a brief solution to the question under discussion.
+In the future, the text of the resolution can be specified as
+link:../../core-modules/cases/[decision] incoming treatment.
+Author of the Article :: Specify the author of the text.
+Approved by:: Select a user of the system that approved the text of the article.
+
+== Categories of Knowledge Base Articles
+
+With a large number of articles, they can be grouped into different categories.
+Categories are created in a separate module *Knowledge Base Categories*.
+
+When creating a category, fill in the following fields.:
+
+Category:: Enter a category name.
+Description:: Enter a description of the category.
+Created::  If necessary, change the user who created the category description category.
+image:CategoriesKB.png[Categories of Knowledge Base Articles]
+
+After creating a category, a list of articles can be added to it. To do this, use the sub-panel
+*Knowledge Base Article* in the Category View Form.
+
+== Managing articles and knowledge base article categories
+
+You can perform the following actions :
+
+*	Sorting the list of records. To do this, click on the icon in the header of the column to be sorted. To reverse sort, click on the icon again.
+*	Editing or deleting information in several records at once,  for this use link:../../introduction/user-interface/record-management/#_mass_updating_records[mass update panel].
+*	View detailed information on the entry, to do this, click on the name of the article or category in the general list.
+*	Data editing, for this purpose or in the Form of viewing, click on the button btn:[Correct],or directly in the List Form, click the button to the left of the record being edited. You can also perform link:../../introduction/user-interface/in-line-editing/[Quick Edit].
+*	Duplication of information, for this in the action menu, select btn:[Duplicate]. Duplication is a convenient way to quickly create similar records, you can change the duplicated information to create a new record.
+*	link:../../introduction/user-interface/record-management/#_merging_records[Merge duplicates], To do ths, select the necessary entries in the List Form and in the action menu select the *Combine* item.
+*	Delete a record, to do this, click on the button btn:[Delete].
+*	Tracking changes to the entered information, to do this, click on the button btn:[View Change Log] in the form of viewing. If the journal needs to change the list of controlled fields - do it in the Studio by setting the parameter link:../../../admin/administration-panel/developer-tools/[*Audit*] corresponding fields.
```

Comment:
I don't see why `Audit` should link to the Developer tools. I suggest linking it to 

`admin/administration-panel/studio/#_fields` 

which is where people need to go to turn on Auditing.